### PR TITLE
add feedliker example

### PR DESCRIPTION
**Pull request by maxkrivich**
solution for `https://github.com/instagrambot/instabot/issues/88`

### CODE REVIEW

File: examples/infinity_feedliker_example.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sys.path.append(os.path.join(sys.path[0], '../'))
+
+from instabot import Bot
+
+bot = Bot()
+bot.login()
+
+while True:
+	bot.like_timeline()
+	time.sleep(1*60)
```
Comment (stasfilin) [inline examples/infinity_feedliker_example.py:21]: 1*60?

you can use:
```python
time.sleep(60)
```

**Review by stasfilin — CHANGES_REQUESTED** [review]
Check my comment

File: examples/infinity_feedliker_example.py (hunk 1 above)
Comment (maxkrivich) [reply examples/infinity_feedliker_example.py:21]: @stasfilin ok, what I must write if I want 2 mins 120 or 2x60, or 3 min 180(3x60) what most user friendly?

Comment (stasfilin) [reply examples/infinity_feedliker_example.py:21]: @maxkrivich if 2 or more mins its okay to use 2*60 or something like this.
Also, if you want to use min, use variable for this, for example

```python
wait = 2*60 # Minutes to seconds
time.sleep(wait)
```
Make sense?

**Comment by stasfilin** [comment]
@maxkrivich Any progress?

**Pull request merged**